spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -764,17 +764,12 @@ def get_connection_holders(self):
 
     @overrides(AbstractAcceptsIncomingSynapses.get_connections_from_machine)
     def get_connections_from_machine(
-            self, transceiver, placement, edge, graph_mapper, routing_infos,
-            synapse_information, machine_time_step, using_extra_monitor_cores,
-            placements=None, monitor_api=None, monitor_cores=None,
-            handle_time_out_configuration=True, fixed_routes=None,
-            extra_monitor=None):
+            self, transceiver, placements, app_edge, graph_mapper,
+            synapse_info, machine_time_step, buffer_manager):
         # pylint: disable=too-many-arguments
         return self.__synapse_manager.get_connections_from_machine(
-            transceiver, placement, edge, graph_mapper, routing_infos,
-            synapse_information, machine_time_step, using_extra_monitor_cores,
-            placements, monitor_api, monitor_cores,
-            handle_time_out_configuration, fixed_routes, extra_monitor)
+            self, transceiver, placements, app_edge, graph_mapper,
```

Comment:
buffer manager go in by top. buffer manager do not go down by bottom. broken

Reply:
I have realised that the buffer manager doesn't work here, so removed.